### PR TITLE
Suspicious LDAP Search Query

### PR DESCRIPTION
**Pull request by ssokolovich**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11553

## Description
New playbook for Core Pack - "Suspicious LDAP Search Query"

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 1]
```diff
@@ -0,0 +1,825 @@
...
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: eddf5d8e-f5e0-4bfa-86f6-1dadfa3c6537
+    type: condition
+    task:
+      id: eddf5d8e-f5e0-4bfa-86f6-1dadfa3c6537
+      version: -1
+      name: Is this a non-legit operation?
+      description: "## Determines if there the LDAP operation was suspicious by the following conditions:\n\n- *Without Host Risk Score:*  \nActor Process name is not Prevalent *AND*\nActor Process Command Line is not Prevalent *AND*\nNo Risk Score for the user\n\n- *With Host Risk Score:*  \n(Actor Process name is not Prevalent *OR*\nActor Process Command Line is not Prevalent) *AND*\nThe Host Risk Score is Medium or High"
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:214]:
```suggestion
      description: "## Determines if the LDAP operation was suspicious by the following conditions:\n\n- *Without Host Risk Score:*  \nActor Process name is not Prevalent *AND*\nActor Process Command Line is not Prevalent *AND*\nNo Risk Score for the user\n\n- *With Host Risk Score:*  \n(Actor Process name is not Prevalent *OR*\nActor Process Command Line is not Prevalent) *AND*\nThe Host Risk Score is Medium or High"
```

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 2]
```diff
@@ -0,0 +1,825 @@
...
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: ba2362bc-345e-417c-8658-9eed09038d85
+    type: regular
+    task:
+      id: ba2362bc-345e-417c-8658-9eed09038d85
+      version: -1
+      name: Close Alert - No results returned
+      description: Close the current alert as False Positive
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:665]:
```suggestion
      description: Close the current alert as False Positive.
```

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+This playbook is designed to handle the following alerts:  
+- Possible LDAP enumeration by unsigned process (medium severity)
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md:2]:
```suggestion
- Possible LDAP enumeration by an unsigned process (medium severity)
```

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+- Suspicious LDAP search query executed (High severity)
+
+The playbook executes the following stages:
+
+Investigation:
+Check the following parameters to determine if remediation actions are needed:
+- Cortex XSIAM alerts related to the hostname by MITRE tactics indicating malicious activity.
+- Whether the Actor Process Command line contains suspicious arguments.
+-  Host risk score is "Medium" or "High".
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md:11]:
```suggestion
- Host risk score is "Medium" or "High".
```

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+* Command-Line Analysis
+
+### Integrations
+
+* CortexCoreIR
+
+### Scripts
+
+* SearchAlertsV2
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md:31]:
```suggestion
Command-Line Analysis

### Integrations

CortexCoreIR

### Scripts

SearchAlertsV2
```

File: Packs/Core/ReleaseNotes/3_0_79.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+
+#### Playbooks
+
+##### New: Suspicious LDAP search query
+
+- New: This playbook is designed to handle the following alerts:
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_79.md:6]:
```suggestion
New: This playbook is designed to handle the following alerts:  
```

File: Packs/Core/ReleaseNotes/3_0_79.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+
+#### Playbooks
+
+##### New: Suspicious LDAP search query
+
+- New: This playbook is designed to handle the following alerts:  
+- Possible LDAP enumeration by unsigned process (medium severity)
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_79.md:7]:
```suggestion
- Possible LDAP enumeration by an unsigned process (medium severity)
```

File: Packs/Core/ReleaseNotes/3_0_79.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+- Suspicious LDAP search query executed (High severity)
+
+The playbook executes the following stages:
+
+Investigation:
+Check the following parameters to determine if remediation actions are needed:
+- Cortex XSIAM alerts related to the hostname by MITRE tactics indicating malicious activity.
+- Whether the Actor Process Command line contains suspicious arguments.
+-  Host risk score is "Medium" or "High".
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_79.md:16]:
```suggestion
- Host risk score is "Medium" or "High".
```

File: Packs/Core/ReleaseNotes/3_0_79.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+Remediation:
+- Handles malicious alerts by terminating the relevant processes.
+- Handles non-malicious alerts identified during the investigation.<~XSIAM> (Available from Cortex XSIAM 2.4).</~XSIAM>
+
+
+#### Triggers Recommendations
+
+##### New: Suspicious LDAP search query
+
+- New: This trigger is responsible for handling the 'Suspicious LDAP search query executed (High severity)' and 'Possible LDAP enumeration by unsigned process (Medium severity)' alerts via the 'Suspicious LDAP search query' playbook<~XSIAM> (Available from Cortex XSIAM 2.4).</~XSIAM>
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_79.md:27]:
```suggestion
New: This trigger is responsible for handling the 'Suspicious LDAP search query executed (High severity)' and 'Possible LDAP enumeration by unsigned process (Medium severity)' alerts via the 'Suspicious LDAP search query' playbook<~XSIAM> (Available from Cortex XSIAM 2.4).</~XSIAM>
```

**Comment by ShirleyDenkberg** [comment]
@OmriItzhak @umishkin Doc review completed.

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 10]
```diff
@@ -0,0 +1,825 @@
...
+    nexttasks:
+      '#none#':
+      - "14"
+    scriptarguments:
+      Commandline:
+        complex:
+          root: alert
+          accessor: cgocmd
+          transformers:
+          - operator: append
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:370]: Please add the 'Unique' transformer, as alert.cgocmd and alert.osparentcmd may have identical values.

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 11]
```diff
@@ -0,0 +1,825 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: 9fcad966-4a26-431d-8436-8b454fe7d3f1
+    type: regular
+    task:
+      id: 9fcad966-4a26-431d-8436-8b454fe7d3f1
+      version: -1
+      name: Get Host Score from Risky Assets
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:531]: Change the task name to "Get Host Risk level"

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 12]
```diff
@@ -0,0 +1,825 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "25":
+    id: "25"
+    taskid: d9bbd834-6027-4bf8-8a9f-5cce15c76d31
+    type: regular
+    task:
+      id: d9bbd834-6027-4bf8-8a9f-5cce15c76d31
+      version: -1
+      name: Get User Score from Risky Assets
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:774]: Change the task name to "Get User Risk level"

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 13]
```diff
@@ -0,0 +1,825 @@
...
+              simple: Medium
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: Core.RiskyUser.risk_level
+            iscontext: true
+          right:
+            value:
+              simple: High
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:296]: please add 'ignorecase: true'

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 14]
```diff
@@ -0,0 +1,825 @@
...
+              simple: "False"
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: Core.AnalyticsPrevalence.Process.value
+            iscontext: true
+          right:
+            value:
+              simple: "False"
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:269]: please add 'ignorecase: true'

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 15]
```diff
@@ -0,0 +1,825 @@
...
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Core.AnalyticsPrevalence.Cmd.value
+            iscontext: true
+          right:
+            value:
+              simple: "False"
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:244]: please add 'ignorecase: true'

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 16]
```diff
@@ -0,0 +1,825 @@
...
+    - label: yes- without Risky Host
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Core.AnalyticsPrevalence.Process.value
+            iscontext: true
+          right:
+            value:
+              simple: "False"
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:236]: please add 'ignorecase: true'

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 17]
```diff
@@ -0,0 +1,825 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 1ac45627-3d10-47b4-8c96-2a4010f239c6
+    type: title
+    task:
+      id: 1ac45627-3d10-47b4-8c96-2a4010f239c6
+      version: -1
+      name: Suspicious Activity Detected
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:46]: Change the task name to "Remediation"

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 18]
```diff
@@ -0,0 +1,825 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: 35be7061-a4df-4ada-8f60-bba6e556c3b7
+    type: title
+    task:
+      id: 35be7061-a4df-4ada-8f60-bba6e556c3b7
+      version: -1
+      name: No suspicious activity detected
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:703]: Change the task name to 'No Results Found'

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 19]
```diff
@@ -0,0 +1,825 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: c1f72153-94cd-4322-8f42-2c33e6c320e6
+    type: regular
+    task:
+      id: c1f72153-94cd-4322-8f42-2c33e6c320e6
+      version: -1
+      name: Terminate the Causality Process Manually
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:629]: change the task name to "Terminate Process Manually"

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 20]
```diff
@@ -0,0 +1,825 @@
...
+      description: |
+        Dear Analyst,
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:631]: Please update the description to:

Dear Analyst,

During the remediation process, the playbook couldn’t terminate the process: ${alert.cgoname}

Please terminate the process manually if possible.

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 21]
```diff
@@ -0,0 +1,825 @@
...
+            value:
+              simple: Core.RiskyUser.risk_level
+            iscontext: true
+          right:
+            value:
+              simple: High
+        - operator: isEqualString
+          left:
+            value:
+              simple: Core.RiskyUser.risk_level
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:300]: please remove the condition  user risk score Equals 'Medium'

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 22]
```diff
@@ -0,0 +1,825 @@
...
+    nexttasks:
+      '#default#':
+      - "4"
+      Yes - With Risky Host:
+      - "19"
+      yes- without Risky Host:
+      - "18"
+    separatecontext: false
+    conditions:
+    - label: yes- without Risky Host
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:227]: change it to 'Yes - without Risky Score'

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 23]
```diff
@@ -0,0 +1,825 @@
...
+      description: "## Determines if there the LDAP operation was suspicious by the following conditions:\n\n- *Without Host Risk Score:*  \nActor Process name is not Prevalent *AND*\nActor Process Command Line is not Prevalent *AND*\nNo Risk Score for the user\n\n- *With Host Risk Score:*  \n(Actor Process name is not Prevalent *OR*\nActor Process Command Line is not Prevalent) *AND*\nThe Host Risk Score is Medium or High"
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      Yes - With Risky Host:
+      - "19"
+      yes- without Risky Host:
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:223]: change it to 'Yes - without Risky Score'

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 24]
```diff
@@ -0,0 +1,825 @@
...
+      version: -1
+      name: Is this a non-legit operation?
+      description: "## Determines if there the LDAP operation was suspicious by the following conditions:\n\n- *Without Host Risk Score:*  \nActor Process name is not Prevalent *AND*\nActor Process Command Line is not Prevalent *AND*\nNo Risk Score for the user\n\n- *With Host Risk Score:*  \n(Actor Process name is not Prevalent *OR*\nActor Process Command Line is not Prevalent) *AND*\nThe Host Risk Score is Medium or High"
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      Yes - With Risky Host:
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:221]: change it to 'Yes - with Risky Score'

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 25]
```diff
@@ -0,0 +1,825 @@
...
+          right:
+            value:
+              simple: "False"
+      - - operator: isEmpty
+          left:
+            value:
+              simple: Core.RiskyHost.risk_level
+            iscontext: true
+          ignorecase: true
+    - label: Yes - With Risky Host
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:251]: change it to 'Yes - with Risky Score'

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 26]
```diff
@@ -0,0 +1,825 @@
+id: Suspicious LDAP search query
+version: -1
+name: Suspicious LDAP search query
+description: "This playbook is designed to handle the following alerts:  \n- Possible LDAP enumeration by unsigned process (medium severity)\n- Suspicious LDAP search query executed (High severity)\n\nThe playbook executes the following stages:\n\nInvestigation:\nCheck the following parameters to determine if remediation actions are needed:\n- Cortex XSIAM alerts related to the hostname by MITRE tactics indicating malicious activity.\n- Whether the Actor Process Command line contains suspicious arguments.\n-  Host risk score is \"Medium\" or \"High\".\n\nRemediation:\n- Handles malicious alerts by terminating the relevant processes.\n- Handles non-malicious alerts identified during the investigation."
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:4]: Update the playbook description to include the use of the user risk score.

**Review by OmriItzhak — APPROVED** [review]

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md (hunk 3 above)
Comment (umishkin) [reply Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md:2]: If the mistake is in the alert name we need to keep it as it is in the alert name

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md [hunk 27]
```diff
@@ -0,0 +1,58 @@
+This playbook is designed to handle the following alerts:  
+- Possible LDAP enumeration by unsigned process (medium severity)
```
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md:2]: let's keep only the alert name without the quotes. needs to be aligned with other playbooks for future automation processes and to make it easy for customer to adopt. 
You can say in a separate sentence or in the title if you want that those are severe alerts

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md [hunk 28]
```diff
@@ -0,0 +1,58 @@
...
+- Suspicious LDAP search query executed (High severity)
```
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md:3]: Same

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md [hunk 29]
```diff
@@ -0,0 +1,58 @@
...
+## Playbook Outputs
+
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+
+---
+
+![Suspicious LDAP search query](../doc_files/Suspicious_LDAP_search_query.png)
```
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md:58]: Change the playbook image as we changed a task name in it

File: Packs/Core/ReleaseNotes/3_0_81.md [hunk 30]
```diff
@@ -0,0 +1,27 @@
...
+- Possible LDAP enumeration by unsigned process (medium severity)
+- Suspicious LDAP search query executed (High severity)
+
+The playbook executes the following stages:
+
+Investigation:
+Check the following parameters to determine if remediation actions are needed:
+- Cortex XSIAM alerts related to the hostname by MITRE tactics indicating malicious activity.
+- Whether the Actor Process Command line contains suspicious arguments.
+- Host risk score is "Medium" or "High".
```
Comment (umishkin) [inline Packs/Core/ReleaseNotes/3_0_81.md:16]: missing user risk score

File: Packs/Core/doc_files/Suspicious_LDAP_search_query.png
Comment (umishkin) [inline Packs/Core/doc_files/Suspicious_LDAP_search_query.png:1]: update this after the fixes from the review

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 31]
```diff
@@ -0,0 +1,879 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 337f923f-d491-41ec-8d15-4034943b5796
+    type: condition
+    task:
+      id: 337f923f-d491-41ec-8d15-4034943b5796
+      version: -1
+      name: Is this a non-legit operation?
```
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:217]: This task name doesn't explains to the customer what you are checking
Please rename the task to something like:

Is the host or user risk score high or non prevalent process or cmd?

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 32]
```diff
@@ -0,0 +1,879 @@
...
+      '#default#':
+      - "4"
+      Yes - With Risky Asset Score:
+      - "19"
+      yes - without Risky Asset Score:
+      - "18"
+    separatecontext: false
+    conditions:
+    - label: yes - without Risky Asset Score
+      condition:
```
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:232]: For me - this condition has a bug.

You check: 
Process prevalence = false 
AND
cmd prevalence = false
AND 
Empty user score
AND 
Empty host score

So you have no suspicious parameter here but you still go to remediation

That needs a fix 

Am I wrong?

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 33]
```diff
@@ -0,0 +1,879 @@
...
+                value:
+                  simple: '-'
+              fields:
+                value:
+                  simple: "2"
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: '((mitreattcktechnique:*T1509* or (sourceBrand:TRAPS and categoryname:Malware)) and caseid:'
```
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:190]: Didn't we said yesterday we will check on 2 sub categories?

**Review by umishkin — CHANGES_REQUESTED** [review]
Please fix all the review comments and send me back to final review

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml (hunk 32 above)
Comment (ssokolovich) [reply Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:232]: the idea here is that if the operation is not prevalent from the process name and its cmd - we will go to the terminate process. why do you consider this a bug?

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml (hunk 33 above)
Comment (ssokolovich) [reply Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:190]: no 
checks for agent alerts + the Command and Scripting technique only

File: Packs/Core/Triggers/Trigger_-_Suspicious_LDAP_search_query.json [hunk 34]
```diff
@@ -0,0 +1,47 @@
+{
+  "trigger_id": "c028aba32791548292993fec8cec5af4",
+  "playbook_id": "Suspicious LDAP search query",
+  "suggestion_reason": "Recommended for 'Suspicious LDAP search query executed (High severity)' and 'Possible LDAP enumeration by unsigned process (Medium severity)' alerts",
```
[the remark this answers is not in the document]
Comment (ssokolovich) [reply Packs/Core/Triggers/Trigger_-_Suspicious_LDAP_search_query.json:4]: Done

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml (hunk 31 above)
Comment (ssokolovich) [reply Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:217]: Changed Please review it again

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml [hunk 35]
```diff
@@ -0,0 +1,896 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: b9bbfbad-b31e-4141-820d-447ae8cbb61a
+    type: condition
+    task:
+      id: b9bbfbad-b31e-4141-820d-447ae8cbb61a
+      version: -1
+      name: Any risky asset involved with a non-prevalent action?
```
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:217]:
```suggestion
      name: Any risky asset involved ?
```

at this step you are in a "prevalent" case and not in a "non-prevalent" case
anyway - no need to mention it the task. 

Please don't forget to update the pic after

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml (hunk 32 above)
Comment (umishkin) [reply Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml]: cause you are making unneeded checks

**Review by umishkin — APPROVED** [review]

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml (hunk 35 above)
Comment (ssokolovich) [reply Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml:217]: Updated the name to `Any risky asset involved with at least one non-prevalent action?`

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md (hunk 29 above)
Comment (ssokolovich) [reply Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query_README.md:58]: done

File: Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml (hunk 32 above)
Comment (ssokolovich) [reply Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml]: split this in to 2 conditions

File: Packs/Core/Triggers/Trigger_-_Suspicious_LDAP_search_query.json [hunk 36]
```diff
@@ -0,0 +1,47 @@
+{
+  "trigger_id": "c028aba32791548292993fec8cec5af4",
+  "playbook_id": "Suspicious LDAP search query",
+  "suggestion_reason": "Recommended for 'Suspicious LDAP search query executed' and 'Possible LDAP enumeration by unsigned process' alerts",
+  "description": "This trigger is responsible for handling the 'Suspicious LDAP search query executed (High severity)' and 'Possible LDAP enumeration by unsigned process (Medium severity)' alerts via the 'Suspicious LDAP search query' playbook",
```
Comment (ssokolovich) [inline Packs/Core/Triggers/Trigger_-_Suspicious_LDAP_search_query.json:5]:
```suggestion
  "description": "This trigger is responsible for handling the 'Suspicious LDAP search query executed' and 'Possible LDAP enumeration by unsigned process' alerts via the 'Suspicious LDAP search query' playbook",
```

**Pull request merged**